packages/devtools_app/pubspec.yaml

```diff
@@ -52,9 +52,9 @@ dependencies:
   string_scanner: ^1.1.0
   url_launcher: ^5.0.0
   url_launcher_web: ^0.1.1+6
-  vm_service: ^6.2.0
+  vm_service: ^7.1.0
```

Comment:
don't know if it matters, but the dds vm_service dep is ^7.0.0

Reply:
So ^7.0.0 is the same as '>=7.0.0 <8.0.0'

So the versions are compatible. As our dependency on vm_service is fragile as all changes are breaking due to implementing a class that isn't expected to be an interface, I like making our dependency a bit on the stricter side. `7.1.0` is the latest published version.